### PR TITLE
Event Page Comments

### PR DESCRIPTION
**Pull request by sherrimitchell**
#1135 View Comments in Events List - added comments column to Events Index on Admin Dashboard. Number of comments for each event are displayed with a link to event page (proposal partial).

screen shot 2016-08-29 at 2 30 48 pm

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -264,12 +264,12 @@
 
   # Activate the test mode, useful when no real iChain is present, like in
   # testing and development environments
-  # config.ichain_test_mode = true
+  config.ichain_test_mode = true
```
Comment (differentreality) [inline config/initializers/devise.rb]: Why are you enabling ichain?

File: app/models/room.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 class Room < ActiveRecord::Base
   belongs_to :venue
+
```
Comment (differentreality) [inline app/models/room.rb]: Not needed

**Comment by differentreality** [comment]
Hey @sherrimitchell :-) 

Please note though that we use rspec, so please remove test/ folder and its contents.

File: app/views/layouts/room_events.html.haml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+%html{:xmlns => "http://www.w3.org/1999/html"}
```
Comment (differentreality) [inline app/views/layouts/room_events.html.haml]: How is this file relevant to this PR?

File: config/initializers/devise.rb (hunk 1 above)
Comment (sherrimitchell) [inline config/initializers/devise.rb]: This setting is enabled in the OSEM devise.rb file. I have not changed it.

Comment (differentreality) [inline config/initializers/devise.rb]: In this line you are uncommenting a line, that makes the setting enabled. The setting was not enabled before, which is why we are seeing this line being changed in your PR. 

If you did not intentionally make this change, please change the lines back to their original format.
Find me on IRC, if you have troubles with this.

Comment (sherrimitchell) [inline config/initializers/devise.rb]: I uncommented these lines according to the instructions in the Contributing Guide for Using iChain in Test Mode. I will comment them out.

File: app/views/layouts/room_events.html.haml (hunk 3 above)
Comment (sherrimitchell) [inline app/views/layouts/room_events.html.haml]: Removed.
